src/Core/Resources/Resources.hpp

```diff
@@ -12,11 +12,11 @@ namespace Core {
  *    - default shaders
  *    - ...
  *
- *   @todo ensure that resources are installed or linked in the parent directory of the
- * libRadiumCore dynamic library.
- *
- *   @todo Generalize the locator such that one can find an offsetted directory from a dynamic
- *   library containing an arbitrary symbol. (easy with cpplocate)
+ *  This namespace also contains functions allowing to set and access paths to store data
+ *  generated by Radium applications, plugins or library in a portable way.
+ *  If not set, the default data path is the working directory of the application.
+ *  For Gui::BaseApplication derived applications, and for plugins, the default data path can be set
+ *  using command line parameter and is stored in the application settings.
```

Comment:
Shall we have a snippet to list in the documentation all the options of the CLI ? This snippet could be added in the documentation `Gui::BaseApplication`.

For users of the present class, it is not easy to know where to search where are the options, so we could also add a link here to the aforementioned CLI documentation.